packages/web-components/src/components/va-card/va-card.tsx

```diff
@@ -0,0 +1,41 @@
+import { 
+  Component,
+  Host, 
+  Prop, 
+  h 
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Card
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
+ */
+
+@Component({
+  tag: 'va-card',
+  styleUrl: 'va-card.scss',
+  shadow: true,
+})
+export class VaCard {
+  /**
+   * If `true`, a drop-shadow will be displayed
+   */
+  @Prop() showShadow?: boolean = false;
+
+  render() {
+    const {
+      showShadow
+    } = this;
+
+    const classes = classnames('va-card',  {'show-shadow': showShadow});
+
+    return (
+      <Host>
+        <div class={classes}>
+          <slot name="content"></slot>
```

Comment:
I wonder if we need to name this slot at all. 🤔  If you just used `<slot></slot>`, then the component could be used without needing any other markup around the content.

```
  <va-card show-shadow={showShadow}>
     <p>Example card content</p>
  </va-card>
```

Reply:
you're right, we don't need to name it. i figured this out as i was working on `va-notification`. it was extremely difficult to target content within `<slot name={content}></slot>` rather than just `<slot></slot>`